calico/reference/configure-cni-plugins.mdx

```diff
@@ -248,6 +248,35 @@ you must also run calico/kube-controllers with the policy, profile, and workload
 
 When using `type: k8s`, the {{prodname}} CNI plugin requires read-only Kubernetes API access to the `Pods` resource in all namespaces.
 
+### Enabling Policy Setup Timeout
+
+If you wish for the CNI to delay a new pod from starting until the pod's policy has been programmed to the dataplane,
+then you may set a policy-setup timeout, e.g.
+
+```json
+{
+  "name": "any_name",
+  "cniVersion": "0.1.0",
+  "type": "calico",
+  "ipam": {
+    "type": "calico-ipam"
+    "policy_setup_timeout_seconds": 10
+  }
+}
+```
+
+:::note
+
+A `policy_setup_timeout_seconds` of 0 disables any policy-setup waiting.
+
+:::
+
+:::note
+
+If the pod's policy is not programmed after the number of seconds specified, the CNI will allow the pod to start its containers anyway.
+
+:::
```

Comment:
Let's move this to the intro text, without the note formatting. (I've added to comment above.)